L1Trigger/Phase2L1ParticleFlow/src/egamma/pftkegalgo_ref.cpp

```diff
@@ -30,6 +31,8 @@ l1ct::PFTkEGAlgoEmuConfig::PFTkEGAlgoEmuConfig(const edm::ParameterSet &pset)
       dEtaValues(pset.getParameter<std::vector<double>>("dEtaValues")),
       dPhiValues(pset.getParameter<std::vector<double>>("dPhiValues")),
       trkQualityPtMin(pset.getParameter<double>("trkQualityPtMin")),
+      doCompositeTkEle(pset.getParameter<bool>("doCompositeTkEle")),
+      nCOMPCAND_PER_CLUSTER(pset.getParameter<uint32_t>("nCOMPCAND_PER_CLUSTER")),
```

Comment:
`nCompCandPerCluster` please, otherwise people will complain when it has to go in CMSSW